examples/datasource-basic/README.md

```diff
@@ -1,16 +1,17 @@
-# Grafana Basic Data Source Plugin example
+# Basic Data Source Plugin for Grafana
 
 [![Build](https://github.com/grafana/grafana-starter-datasource/workflows/CI/badge.svg)](https://github.com/grafana/grafana-starter-datasource/actions?query=workflow%3A%22CI%22)
 
-This template is a starting point for building Grafana data source plugins.
+This repository contains a simple data source plugin example.
+## Overview
 
-## What is a Grafana data source plugin?
+The Basic Data Source Plugin offers a streamlined starting point for developers to understand the essential structure and functionality required for adding their own data sources to Grafana.
 
-Grafana supports a wide range of data sources, including Prometheus, MySQL, and even Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana data source plugins enable you to integrate such solutions with Grafana.
+Grafana supports a wide range of data sources, including Prometheus, MySQL, and Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana data source plugins enable you to integrate such solutions with Grafana.
 
 ## Get started
 
-A data source backend plugin consists of both frontend and backend components.
+Data source plugins consist of both frontend and backend components. Install these components with the following CLI commands.
```

Comment:
They don't have to, they can be frontend only. This datasource is a backend one (whether it should be as our basic example is debatable)

Reply:
done